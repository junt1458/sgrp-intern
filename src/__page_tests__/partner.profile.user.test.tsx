import { render } from '@testing-library/react';
import PartnerProfilePage from '../pages/partner/profile';
import { Provider } from 'urql';

jest.mock('next/router', () => ({
  useRouter() {
    return {
      route: '/',
      push: jest.fn(),
    };
  },
}));

it('should render the component', () => {
  const responseState = {
    executeQuery: () => {},
  };
  const { container } = render(
    <Provider value={responseState}>
      <PartnerProfilePage />
    </Provider>
  );
  expect(container).toMatchSnapshot();
});
