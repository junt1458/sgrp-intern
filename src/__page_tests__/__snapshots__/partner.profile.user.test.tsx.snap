// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the component 1`] = `
<div>
  <div
    class="fixed left-0 top-0 z-50 flex h-14 w-full items-center justify-between bg-slate-900 px-4 py-2 text-xl text-white"
  >
    <a
      href="/"
    >
      Internship Application System
    </a>
  </div>
  <div
    class="h-14 w-full py-2"
  />
  <div
    class="my-8"
  >
    <div
      class="flex flex-wrap items-center justify-center"
    >
      <img
        alt="Loading icon"
        height="200"
        src="/loading.gif"
        width="200"
      />
      <div
        class="w-full text-center"
      >
        Loading...
      </div>
    </div>
  </div>
</div>
`;
